importers/yelp.ca/generate_csv.py

```diff
@@ -0,0 +1,127 @@
+import lxml.html as html
+import os
+import json
+import itertools
+from utils import get_text_from_css
+import re
+import csv
+from collections import Counter
+
+
+yelp_base_url = 'https://www.yelp.ca'
+
+
+def get_location_info(location_details_filename):
+    """
+    get_location_info reads html file and returns a
+    which contains information on the location
+    """
+    with open(location_details_filename, 'r') as html_file:
+        content = html_file.read()
+        root = html.fromstring(content)
+        coordinates = root.cssselect('.lightbox-map')[0].xpath('@data-map-state')[0]
+        # xpath pulls attributes from within given div
+        coordinates = json.loads(coordinates)
+        biz_name = get_text_from_css(root,'div.u-space-t1 > h1')
+        phone_number = get_text_from_css(root,'span.biz-phone')
+        latitude = coordinates['center']['latitude']
+        longitude = coordinates['center']['longitude']
+        location_url = root.cssselect('.mapbox-map > a')[0].xpath('@href')[0]
+        # gets map url of location
+        address = get_text_from_css(root,'strong.street-address')
```

Comment:
The address is coming out right next to the city name.  For example, "128 University Avenue WWindsor, ON N9A 5N9"

Please look into getting a space or comma between the two like  "128 University Avenue W Windsor, ON N9A 5N9".

Without the space, software and web services processing the address could mess up and confuse the WWindsor as the correctly spelled name of a city or street.